packages/piro/src/Piro_TempusSolver_Def.hpp

```diff
@@ -951,6 +951,18 @@ setInitialState(Scalar t0,
  
 }
 
+#ifdef ALBANY_BUILD
+template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Node>

```

Comment:
You might be encouraged to note that Tpetra has already begun the process of deprecating all template parameters but Scalar -- a plan that we first documented in January on Trilinos' wiki.